src/mnode/mnode-rpc.cpp

```diff
@@ -1216,159 +1217,184 @@ UniValue governance(const UniValue& params, bool fHelp)
     return NullUniValue;
 }
 
+/**
+ * pastelid RPC command.
+ * 
+ * \param params - RPC command parameters
+ * \param fHelp - true to show pastelid usage
+ * \return univalue result object
+ */
 UniValue pastelid(const UniValue& params, bool fHelp)
 {
-    RPC_CMD_PARSER(PASTELID, params, newkey, importkey, list, sign, sign_by_key, verify);
+    RPC_CMD_PARSER(PASTELID, params, newkey, importkey, list, sign, sign__by__key, verify);
 
     if (fHelp || !PASTELID.IsCmdSupported())
         throw runtime_error(
 R"(pastelid "command"...
-Set of commands to deal with PatelID and related actions
-    PastelID is the base58-encoded public key of the EdDSA448 key pair. EdDSA448 public key is 57 bytes
+Set of commands to deal with PastelID and related actions
+PastelID is the base58-encoded public key of the EdDSA448 key pair. EdDSA448 public key is 57 bytes
 
 Arguments:
 1. "command"        (string or set of strings, required) The command to execute
 
 Available commands:
-  newkey "passphrase"                        - Generate new PastelID and associated keys (EdDSA448). Return PastelID base58-encoded
-                                               "passphrase" will be used to encrypt the key file
-  importkey "key" <"passphrase">             - Import private "key" (EdDSA448) as PKCS8 encrypted string in PEM format. Return PastelID base58-encoded
-                                               "passphrase" (optional) to decrypt the key for the purpose of validating and returning PastelID
-  											   NOTE: without "passphrase" key cannot be validated and if key is bad (not EdDSA448) call to "sign" will fail
-  list                                       - List all internally stored PastelID and keys.
-  sign "text" "PastelID" "passphrase"        - Sign "text" with the internally stored private key associated with the PastelID.
-  sign_by_key "text" "key" "passphrase"      - Sign "text" with the private "key" (EdDSA448) as PKCS8 encrypted string in PEM format.
-  verify "text" "signature" "PastelID"       - Verify "text"'s "signature" with the PastelID.
+  newkey "passphrase"                                - Generate new PastelID, associated keys (EdDSA448) and LegRoast signing keys.
+                                                       Return PastelID and LegRoast signing public key base58-encoded.
+                                                       "passphrase" will be used to encrypt the key file.
+  importkey "key" <"passphrase">                     - Import private "key" (EdDSA448) as PKCS8 encrypted string in PEM format. Return PastelID base58-encoded
+                                                       "passphrase" (optional) to decrypt the key for the purpose of validating and returning PastelID.
+  											           NOTE: without "passphrase" key cannot be validated and if key is bad (not EdDSA448) call to "sign" will fail
```

Comment:
NOTE offset

Reply:
Forgot about this one, will correct in the next checkin...